caches_bench/caches_gc_overhead_comparison.go

```diff
@@ -14,7 +14,7 @@ func gcPause() time.Duration {
 	runtime.GC()
 	var stats debug.GCStats
 	debug.ReadGCStats(&stats)
-	return stats.Pause[0]
+	return stats.PauseTotal
```

Comment:
This change makes our all benchmarks wrong :(
Using the latest GC pause was done on purpose. Here we get gc pause for the whole application so in current code, GC pause for Map will be always higher than GC pause for BigCache since it will sum real map gc pause and bigcache gc pause. To prevent this we should subtract previously observed GC Pause.

Reply:
https://github.com/allegro/bigcache-bench/issues/7